flag.go

```diff
@@ -528,6 +529,7 @@ func (f *FlagSet) parseShortArg(s string, args []string) (a []string, err error)
 }
 
 func (f *FlagSet) parseArgs(args []string) (err error) {
+Loop:
```

Comment:
This is not really needed, there's only a single loop so a bare `break` should do it.